pkg/cmd/create/create_codeship.go

```diff
@@ -1,468 +0,0 @@
-
-	cmd.Flags().StringVarP(&options.Flags.ForkOrganisationGitRepo, "fork-git-repo", "f", kube.DefaultOrganisationGitRepoURL, "The Git repository used as the fork when creating new Organisation Git repos")
-
-	cmd.Flags().StringVarP(&options.Flags.GitUser, "git-user", "", "Codeship", "The name to use for any git commits")
-	cmd.Flags().StringVarP(&options.Flags.GitEmail, "git-email", "", "codeship@jenkins-x.io", "The email to use for any git commits")
-
-	cmd.Flags().StringVarP(&options.Flags.GKEServiceAccount, "gke-service-account", "", "", "The GKE service account to use")
-}
-
-func (o *CreateCodeshipOptions) validate() error {
-	if o.Flags.OrganisationName == "" {
-		return errors.New("No organisation has been set")
-	}
-
-	// TODO we should only do this if a GKE cluster has been specified
-	if o.Flags.GKEServiceAccount == "" {
-		return errors.New("No GKE service account has been set")
-	}
-	return nil
-}
-
-// Run implements this command
-func (o *CreateCodeshipOptions) Run() error {
-	surveyOpts := survey.WithStdio(o.In, o.Out, o.Err)
-	if !o.Flags.SkipLogin {
-		err := o.RunCommandVerbose("gcloud", "auth", "login", "--brief")
-		if err != nil {
-			return err
-		}
-	}
-
-	if o.Flags.OrganisationName == "" {
-		o.Flags.OrganisationName = strings.ToLower(randomdata.SillyName())
-	}
-
-	// if gke-service-account is not set, create the service account
-	if o.Flags.GKEServiceAccount == "" {
-		gkeServiceAccountPath := path.Join(util.HomeDir(), fmt.Sprintf("%s.key.json", o.Flags.OrganisationName))
-
-		o.CreateGkeServiceAccountOptions.Flags.Name = o.Flags.OrganisationName
-		o.CreateGkeServiceAccountOptions.CommonOptions.BatchMode = o.CreateOptions.CommonOptions.BatchMode
-		o.CreateGkeServiceAccountOptions.Flags.SkipLogin = true
-		err := o.CreateGkeServiceAccountOptions.Run()
-		if err != nil {
-			return err
-		}
-
-		o.Flags.GKEServiceAccount = gkeServiceAccountPath
-	}
-
-	err := o.validate()
-	if err != nil {
-		return err
-	}
-
-	if o.Flags.CodeshipUsername == "" {
-		prompt := &survey.Input{
-			Message: "Username for Codeship",
-			Help:    "This will not be stored anywhere",
-		}
-
-		err := survey.AskOne(prompt, &o.Flags.CodeshipUsername, nil, surveyOpts)
-		if err != nil {
-			return err
-		}
-	}
-
-	if o.Flags.CodeshipPassword == "" {
-		prompt := &survey.Password{
-			Message: "Password for Codeship",
-			Help:    "This will not be stored anywhere",
-		}
-
-		err := survey.AskOne(prompt, &o.Flags.CodeshipPassword, nil, surveyOpts)
-		if err != nil {
-			return err
-		}
-	}
-
-	if o.Flags.CodeshipOrganisation == "" {
-		prompt := &survey.Input{
-			Message: "Codeship organisation",
-			Help:    "This will not be stored anywhere",
-		}
-
-		err := survey.AskOne(prompt, &o.Flags.CodeshipOrganisation, nil, surveyOpts)
-		if err != nil {
-			return err
-		}
-	}
-
-	organisationDir, err := util.OrganisationsDir()
```

Comment:
`util.OrganisationsDir` is probably a candidate for later removal. It is used by a couple of other commands, but I think this will all go. 

We want to make sure to scan the command files like this one to check for potentially unused code.